ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.hpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <functional>
+#include <optional>
+#include "tensor/tensor.hpp"
+#include "third_party/magic_enum/magic_enum.hpp"
+
+namespace ttnn::operations::unary_backward {
+
+constexpr uint8_t DefaultQueueId = 0;
+enum class UnaryBackwardOpType {
+    MUL_BW,
+    CLAMP_MIN_BW,
+    CLAMP_BW,
+    ASSIGN_BW,
+    MULTIGAMMALN_BW,
+    ADD_BW,
+    EQ_BW,
+};
+
+struct UnaryBackwardFunction{
```

Comment:
why is this in a struct?
why do you use std::function here?
Have you seen @eyonland comment in the MCW channel on it?

Reply:
Yes @ayerofieiev-tt , we are currently working on that struct. we are planning to resolve that on a seperate PR. Until then the plna is to migrate the ops as how it is done.